templates/expo-template-bare-minimum/ios/HelloWorld.xcodeproj/project.pbxproj

```diff
@@ -214,7 +214,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "export NODE_BINARY=node\n../node_modules/react-native/scripts/react-native-xcode.sh\n";
+			shellScript = "export NODE_BINARY=node\n`node --print \"require.resolve('react-native/package.json').slice(0, -13) + '/scripts/react-native-xcode.sh'\"`\n";
```

Comment:
```suggestion
			shellScript = "export NODE_BINARY=node\n`node --print \"require('path').dirname(require.resolve('react-native/package.json'))\"`/scripts/react-native-xcode.sh\n";
```

Reply:
just a nit for `slice(0, -13)` without comments here. how do you think if this getting clearer?
`require('path').dirname(require.resolve('react-native/package.json'))`

on the other hand, there is a semantic difference of original fix where placing react-native-xcode.sh inside backquotes. it would be if we can use keep original semantic 😅

Reply:
good feedback 👍 updated it and i think it's much clearer to use `.dirname`